plugins/nextcloud-rule-exclusions-before.conf

```
@@ -799,6 +799,37 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/(?:[^/]+/){2}[
         "t:none,\
         ctl:ruleRemoveById=950100"
 
+# Deleting a contact from addressbook
+# XML parser fails because a content type is set without a request body
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/[^/]+/[^/]+\.vcf$" \
+    "id:9508324,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
+    chain"
+    SecRule REQUEST_METHOD "@streq DELETE" \
+        "t:none,\
+        chain"
+        SecRule REQUEST_BODY_LENGTH "@eq 0" \
+            "t:none,\
+            ctl:ruleRemoveById=200002"
+
+# Syncing contacts via a dav client
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/(?:[^/]+/)?$" \
```

Same here, keep it if you like.
```suggestion
SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/(?:[^/]+/){1,2}$" \
```